isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -486,7 +491,8 @@ TEST_F(ObservationPair, FunctionalTestJigsawCamSolveAll) {
   QString outCnetFileName = prefix.path() + "/outTemp.net";
   QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+cnetPath, "onet="+outCnetFileName,
                            "observations=yes", "update=yes", "Cksolvedegree=3",
-                           "Camsolve=all", "twist=no", "Spsolve=none", "Radius=no", "imagescsv=on", "file_prefix="+prefix.path()+"/"};
+                           "Camsolve=all", "twist=no", "Spsolve=none", "Radius=no", "imagescsv=on", "file_prefix="+prefix.path()+"/",
+                           "adjustment_output=adj_temp.h5"};
```

Comment:
Were these tests failing because it was looking for adjustment_output? I would assume that this parameter should be optional. Otherwise, this is a breaking change.

Reply:
This can be changed to optional.